jetty-ee10/jetty-ee10-tests/jetty-ee10-test-integration/src/test/resources/RFC2616Base.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0"?>
-<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://www.eclipse.org/jetty/configure_9_3.dtd">
+<?xml version="1.0"?><!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://www.eclipse.org/jetty/configure_10_0.dtd">
```

Comment:
Again, these changes are extraneous to the deployment manager - better to have raised a separate issue.

Reply:
The whitespace changes is are unfortunate.
But they are automatic with the IDE if you edit the file to make it conform to Jetty Syntax rules.
Just turn of the whitespace differences in your diff/pull-request review.